rando.py

```diff
@@ -108,3 +136,9 @@ def pick(self, mess, args):
     def eightball(self, mess, args):
         """Returns a Magic 8-Ball reply"""
         return random.choice(EIGHTBALL_ANSWERS)
+
+    @botcmd(split_args_with=' ', admin_only=False)
+    def shuffle(self, mess, args):
```

Comment:
That's a neat idea! I wish I had included that command from the start 😅

Reply:
Also, would you mind adding this to the Commands section of the README: https://github.com/samueldg/err-rando#commands ?

This is worth socializing!